src/main/java/com/synopsys/integration/detect/tool/signaturescanner/BlackDuckSignatureScanner.java

```diff
@@ -221,7 +225,7 @@ protected ScanBatchBuilder createDefaultScanBatchBuilder(NameVersion projectName
         String suffix = signatureScannerOptions.getCodeLocationSuffix().orElse(null);
 
         for (SignatureScanPath scanPath : signatureScanPaths) {
-            String codeLocationName = codeLocationNameManager.createScanCodeLocationName(sourcePath, scanPath.getTargetPath(), dockerTarFile, projectName, projectVersionName, prefix, suffix);
+            String codeLocationName = codeLocationNameManager.createScanCodeLocationName(sourcePath, scanPath.getTargetPath(), dockerTargetData.getSquashedImage().orElse(null), projectName, projectVersionName, prefix, suffix);
```

Comment:
Why doesn't this consider the other tar files present?

Reply:
it should default to provided tar before null, you're right